app/views/shared/_csv_import_modal.html.erb

```diff
@@ -25,7 +25,7 @@
                 <li>
                   <%= f.file_field :file, accept: 'text/csv', class: 'form-control-file', style: "margin: auto;"  %>
                 </li>
-                <li style="margin-bottom: 27px;">Then click the "Import CSV" button <br>to import your storage locations.</li>
+                <li style="margin-bottom: 27px;"><%= "Then click the \"Import CSV\" button to import your #{self.controller_name.gsub("_", " ")}." %></li>

```

Comment:
I see what you're doing here, but I'm not sure relying on the controller name is the best way. It looks like each of these has a `title` already passed in (like https://github.com/rubyforgood/human-essentials/blob/main/app/views/product_drive_participants/index.html.erb#L79), maybe you can do it the same way.

I notice that all of the titles start with "Import". Maybe you could re-use that? To be slightly fancy, specifically you could remove "Import" from the titles passed in (from each of the files) and then put the word "Import" in the modal view. Then you could use `<%= title %>` as part of the text here.

Reply:
.... also ... are we being too fancy here anyway? We could change it to `Then click the "Import CSV" button.` or `Then click the "Import CSV" button to import your records` as another idea.

Reply:
Hey. I addresed your comments here [b5e2448](https://github.com/rubyforgood/human-essentials/pull/3492/commits/b5e2448b53dd8fb0debcc86ca91e0b102bceaa7a)